docs/pages-for-subheaders/installation-requirements.md

```diff
@@ -55,68 +55,126 @@ If you are installing Rancher on a K3s cluster with **Raspbian Buster**, follow
 
 If you are installing Rancher on a K3s cluster with Alpine Linux, follow [these steps](https://rancher.com/docs/k3s/latest/en/advanced/#additional-preparation-for-alpine-linux-setup) for additional setup.
 
-### RKE2 Specific Requirements
+### RKE Specific Requirements
 
-For the container runtime, RKE2 bundles its own containerd. Docker is not required for RKE2 installs.
+For the container runtime, RKE should work with any modern Docker version.
```

Comment:
We have validated Docker versions for RKE provisioned clusters in the support matrix. Should this be called out instead of any modern? https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-7-6/

Reply:
Addressed as of 421e12c05e9